src/transactions/feed/TransactionDetailsScreen.tsx

```diff
@@ -189,8 +187,7 @@ function TransactionDetailsScreen({ navigation, route }: Props) {
             >
               <View style={styles.rowContainer}>
                 <Text style={styles.blockExplorerLink}>
-                  {transactionNetwork === Network.Celo && t('viewOnCeloScan')}
-                  {transactionNetwork === Network.Ethereum && t('viewOnEthereumBlockExplorer')}
+                  {t(TX_EXPLORER_LINK_TRANSLATION_STRINGS[transaction.networkId])}
```

Comment:
The problem with this pattern is it will prevent static extraction of translatable strings.

I admit we haven't been using it yet, but would be nice not to break this now.
Or have dicuss it explicitely if we do not want to pursue this goal.

Reply:
thanks, addressed this in https://github.com/valora-inc/wallet/pull/4887